packages/ng/fancy-box/fancy-box.component.ts

```diff
@@ -0,0 +1,35 @@
+import { ChangeDetectionStrategy, Component, HostBinding, Input, ViewEncapsulation } from '@angular/core';
+
+@Component({
+	selector: 'lu-fancy-box',
+	standalone: true,
+	templateUrl: './fancy-box.component.html',
+	styleUrls: ['./fancy-box.component.scss'],
+	changeDetection: ChangeDetectionStrategy.OnPush,
+	encapsulation: ViewEncapsulation.None,
+	host: {
+		class: 'fancyBox',
+	},
+})
+export class FancyBoxComponent {
+	@Input()
+	foreground: string = null;
```

Comment:
```suggestion
	foreground?: string;
```

Because it's typed as string, using a `?:` operator for type and not setting null makes it more efficient.